webrtc.html

```diff
@@ -2087,19 +2126,7 @@
               </li>
 
               <li>
-                <p>If <var>connection</var>'s <a href=
-                "#dom-peerconnection-signaling-state"><code>RTCPeerConnection</code>
-                signalingState</a> is <code>stable</code>, then fire an <a href=
-                "#event-negotiation">negotiationneeded</a> event at
-                <var>connection</var>.</p>
-
-                <div class="note">
-                  <p>Throughout this document, we need to indicate what should
-                  happen if a change happens and the state is not stable, most
-                  likely to fire the negotiationneeded event when the state
-                  eventually returns to stable.
```

Comment:
Updated setLocal. setRemote already refers to setLocal's processing algorithm, so no action needed there.

I don't see much value in the app inspecting n_t_f_n_n, so I didn't go down that road in this PR.